core/spec/models/spree/order_spec.rb

```diff
@@ -326,13 +326,13 @@ def compute(computable)
     let(:order_2) { Spree::Order.create }
 
     it "destroys the other order" do
-      order_1.merge!(order_2)
+      Spree::OrderMerge.new(order_1, order_2)
```

Comment:
I'm thinking we should move this whole #marge! context into it's own order_merger_spec.rb